proto/panacea/market/v2/deal.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+package panacea.market.v2;
+
+option go_package = "github.com/medibloc/panacea-core/x/market/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+// Deal defines a deal.
+// TODO: Could remove complete_data_count field
+message Deal {
+  uint64 deal_id = 1;
+  string deal_address = 2;
+  repeated string data_schema = 3;
+  cosmos.base.v1beta1.Coin budget = 4;
+  repeated string trusted_data_validators = 5;
+  uint64 target_num_data = 6;
+  uint64 filled_num_data = 7;
```

Comment:
Aha. Now I understand what this field means. I guess you're gonna update this value whenever you get a SellData tx. 
Then, I think the `cur num data` and `max num data` would be clearer. wdyt?
it's little awkward that the deal is 'filled'. 

btw. i'm not sure yet how we'll improve this deal concept in near future. But, from my gut feeling, it would be enough to have only budget and pricePerData. Then, the deal becomes deactivated when the balance of the deal account runs out.
But, don't think too much of this now. let's see when we implement SellData tx

Reply:
I agree with your review that rename the field to ```cur_num_data``` and ```max_num_data```.  Also we need to think this when we implement SellData feature. And It's good idea that we just have ```budget``` field and ```pricePerData``` field! Thank you for your review.

Reply:
Good! `max_num_data` and `cur_num_data` are clearer.